### DISCUSSION TITLE
MacOS Embedded MongoDB Issue

### DISCUSSION THREAD

**Discussion by v-shalimov** [quarkusio/quarkus#25093] [Q&A] [answered]
Hi All!

I am developing using Quarkus on MacBook Pro and wanted to upgrade to the latest version (2.8.1-Final). After upgrade all tests failed with an error:

```
Extract /Users/vshalimov/.embedmongo/tools/db/mongodb-database-tools-macos-x86_64-100.5.1.zip START
2022-04-22 13:00:03,524 WARN  [de.fla.emb.pro.ext.AbstractExtractor] (main) 
--------------------------
If you get this exception more than once, you should check if the file is corrupt.
If you remove the file (/Users/vshalimov/.embedmongo/tools/db/mongodb-database-tools-macos-x86_64-100.5.1.zip), it will be downloaded again.
--------------------------: java.io.IOException: Input is not in the .gz format
```

Will be very thankful for any assistance in that regard.

[a comment by an automated account is not shown]

**Comment by loicmathieu** [comment, marked as answer]
I'm surprised it tries to download Mongo 5.1! 
Anyway this seems like an issue on flapdoodle embedded MongoDB that download a ZIP and expect an tar.gz, maybe it's better to ask on their side.

**Comment by v-shalimov** [comment]
OK. Thank you for an assistance!
